pkg/plugins/utils/truncate/main.go

```diff
@@ -0,0 +1,16 @@
+package truncate
+
+import "unicode/utf8"
+
+// String safely truncate a string
+func String(str string, length int) string {
+	if length <= 0 {
+		return ""
+	}
+
+	if utf8.RuneCountInString(str) < length {
+		return str
+	}
+
+	return string([]rune(str)[:length])
```

Comment:
I would add `...` at the end of the returned string to indicate it has been has been truncated.

Reply:
Indeed that could be an nice to have. 

So instead of "This line is too long", we would have "This line is too l..."